### PR TITLE
feat(process): get angle from image

### PR DESCRIPTION
**Pull request by Shyrx**

### CODE REVIEW

File: src/process/process.cc [hunk 1]
```diff
@@ -155,14 +163,32 @@ namespace blind
...
+            getDirection(nb_labels, centroids, yellow_cones, stats);
+
+        cv::arrowedLine(original, base, mid, cv::Scalar(255, 255, 255));
+
+        return original;
+    }
+
+    int get_angle(cv::Mat &img)
+    {
+        cvtColor(img, img, cv::COLOR_BGR2HSV);
```
Comment (Cobaltarena) [inline src/process/process.cc:176]: Isnt this already computed in a previous step ?

Comment (Shyrx) [reply src/process/process.cc:176]: No as those are two separate function, but we could factorize

File: src/process/process.cc [hunk 2]
```diff
@@ -8,6 +8,8 @@
 #include <opencv2/core/types.hpp>
 #include <opencv2/imgproc.hpp>
 
+#define PI 3.14159265358979323846
```
Comment (TristanFloch) [inline src/process/process.cc:11]: Probably cleaner to use a constexpr

**Review by TristanFloch — APPROVED** [review]

File: src/process/process.cc (hunk 1 above)
Comment (Cobaltarena) [reply src/process/process.cc:176]: Osef au pire

File: src/process/process.cc (hunk 2 above)
Comment (Cobaltarena) [reply src/process/process.cc]: Done

**Review by Cobaltarena — APPROVED** [review]

**Review by TristanFloch — APPROVED** [review]

**Review by Cobaltarena — APPROVED** [review]

**Pull request merged**